pychunkedgraph/meshing/meshgen.py

```diff
@@ -30,68 +30,68 @@ def _get_sv_to_node_mapping_internal(cg, chunk_id, unbreakable_only):
     xyz_plus_chunk_min_node_id = cg.get_chunk_id(
         layer=1, x=x + 1, y=y + 1, z=z + 1)
 
-    row_keys = ['atomic_partners', 'connected']
+    columns = [column_keys.Connectivity.Partner,
+               column_keys.Connectivity.Connected]
     if unbreakable_only:
-        row_keys.append('affinities')
+        columns.append(column_keys.Connectivity.Affinity)
 
-    seg_ids_center = cg.range_read_chunk(1, x, y, z, row_keys=row_keys)
+    seg_ids_center = cg.range_read_chunk(1, x, y, z, columns=columns)
 
     seg_ids_face_neighbors = {}
     seg_ids_face_neighbors.update(
-        cg.range_read_chunk(1, x + 1, y, z, row_keys=row_keys))
+        cg.range_read_chunk(1, x + 1, y, z, columns=columns))
     seg_ids_face_neighbors.update(
-        cg.range_read_chunk(1, x, y + 1, z, row_keys=row_keys))
+        cg.range_read_chunk(1, x, y + 1, z, columns=columns))
     seg_ids_face_neighbors.update(
-        cg.range_read_chunk(1, x, y, z + 1, row_keys=row_keys))
+        cg.range_read_chunk(1, x, y, z + 1, columns=columns))
 
     seg_ids_edge_neighbors = {}
     seg_ids_edge_neighbors.update(
-        cg.range_read_chunk(1, x + 1, y + 1, z, row_keys=row_keys))
+        cg.range_read_chunk(1, x + 1, y + 1, z, columns=columns))
     seg_ids_edge_neighbors.update(
-        cg.range_read_chunk(1, x, y + 1, z + 1, row_keys=row_keys))
+        cg.range_read_chunk(1, x, y + 1, z + 1, columns=columns))
     seg_ids_edge_neighbors.update(
-        cg.range_read_chunk(1, x + 1, y, z + 1, row_keys=row_keys))
+        cg.range_read_chunk(1, x + 1, y, z + 1, columns=columns))
 
     # Retrieve all face-adjacent supervoxel
     one_hop_neighbors = {}
-    for seg_id_b, data in seg_ids_center.items():
-        data = cg.flatten_row_dict(data.cells['0'])
-        partners = data['atomic_partners']
-        connected = get_connected(data['connected'])
+    for seg_id, data in seg_ids_center.items():
+        data = cg.flatten_row_dict(data)
+        partners = data[column_keys.Connectivity.Partner]
+        connected = data[column_keys.Connectivity.Connected]
```

Comment:
I do not quite understand how this can work when you are not using `get_connected` anymore